### PR TITLE
Rtl fix

### PR DESCRIPTION
**Pull request by tasnim0tantawi**
RTL languages should make the some parts of UI displayed from right to left.

 image

Most the edits I did:
`
 style={isRtl(i18n.language) ? { direction: "rtl" } : {}}
`
Which is changing HTML dir to rtl if the language is one of the known 5 rtl languages.

### CODE REVIEW

File: src/components/EditorHeader/ControlPanel.jsx [hunk 1]
```diff
@@ -1374,15 +1376,20 @@ export default function ControlPanel({
...
+      <div
+        className="py-1.5 px-5 flex justify-between items-center rounded-xl my-1 sm:mx-1 xl:mx-6 select-none overflow-hidden toolbar-theme"
+        style={isRtl(i18n.language) ? { direction: "rtl" } : {}}
+      >
         <div className="flex justify-start items-center">
           <LayoutDropdown />
           <Divider layout="vertical" margin="8px" />
           <Dropdown
             style={{ width: "240px" }}
             position="bottomLeft"
```
Comment (1ilit) [inline src/components/EditorHeader/ControlPanel.jsx:1388]: Could you please add this to `position` just make the dropdowns all consistent

```js
isRtl(i18n.language) ? "bottomRight" : "bottomLeft"
```

**Review by 1ilit — CHANGES_REQUESTED** [review]
Idk if this absolutely has to be flipped or not but it makes the type pretty difficult to read 
image

Maybe we can force it to be 'ltr' all the time. But then it will be inconsistent with the 'type' of Type fields. Let me know what you think.
https://github.com/drawdb-io/drawdb/blob/2f1cca13d6a9e8dc232a53a894849271ba073c10/src/components/EditorSidePanel/TablesTab/TableField.jsx#L52

<br/>

Also this breaks the resizing of the side panel so please add this to `handleResize`
```js
    const w = isRtl(i18n.language) ? window.innerWidth - e.clientX : e.clientX;
```
here:
https://github.com/drawdb-io/drawdb/blob/2f1cca13d6a9e8dc232a53a894849271ba073c10/src/components/Workspace.jsx#L56 

Other than that lgtm

File: src/components/EditorHeader/ControlPanel.jsx (hunk 1 above)
Comment (tasnim0tantawi) [reply src/components/EditorHeader/ControlPanel.jsx]: @1ilit Thanks so much for the code review, I've learned things! Please check my latest commit. I have addressed the issues you mentioned. Also, I made both field name and data type ltr as they will always be in English.
image

**Review by 1ilit — APPROVED** [review]
Thank you!

**Pull request merged**